pkg/ccr/base/spec.go

```
@@ -1131,6 +1131,9 @@ func (s *Spec) WaitTransactionDone(txnId int64) {
 	for {
 		if err := s.waitTransactionDone(txnId); err != nil {
 			log.Errorf("wait transaction done failed, err +%v", err)
+			if strings.Contains(err.Error(), "does not exist") {
+				break
+			}
```

Skipping the waiting process by default is not recommended, as it may obscure critical issues. Could you implement a feature flag that is disabled by default?